frontend/src/components/Mobile/TopNavBar.tsx

```diff
@@ -24,9 +26,9 @@ function TopNavBar({
   rightComponent,
 }: {
   title: string
-  leftIcon?: true
+  leftIcon?: boolean
   leftIconComponent?: ReactElement
-  rightComponent?: ReactElement
+  rightComponent?: ReactElement | typeof ReactComponent
```

Comment:
hmm is ReactComponent type needed? 
- whenever someone uses right-component, likely will have a `onClick`attatched to it -> should be an ReactElement.

Reply:
Resolve in commit 5d3a88d93cd2292ee86164198444021046833cc7